packages/overlay/src/overlay-trigger.ts

```diff
@@ -55,25 +61,42 @@ export class OverlayTrigger extends LitElement {
     public onOverlayOpen(event: Event, interaction: TriggerInteractions): void {
         const isClick = interaction === 'click';
         const overlayElement = isClick ? this.clickContent : this.hoverContent;
-        const delayAttribute = overlayElement
-            ? overlayElement.getAttribute('delay')
-            : null;
-        const delay = delayAttribute ? parseFloat(delayAttribute) : 0;
 
+        /* istanbul ignore if */
         if (!overlayElement) {
             return;
         }
+        if (!overlayRoot) {
+            overlayRoot = new OverlayRoot();
+        }
+
+        const delayAttribute = overlayElement.getAttribute('delay');
+        const delay = delayAttribute ? parseFloat(delayAttribute) : 0;
+
+        const queryThemeDetail: ThemeData = {
+            color: undefined,
+            size: undefined,
+        };
+        const queryThemeEvent = new CustomEvent<ThemeData>('query-theme', {
+            bubbles: true,
+            composed: true,
+            detail: queryThemeDetail,
+            cancelable: true,
```

Comment:
I initially set this as `cancelable` so that it could merge with the `overlayOpenEvent` below and make a single traversal, but I ended up not wanting to mess with the data structure there...I could see merging them again, or moving to a `stopPropagation()` approach instead...thoughts?